### DISCUSSION TITLE
Benefits Not Recieved After Approved Student

### DISCUSSION THREAD

**Discussion by sigfriedherzog001** [community/community#140741] [GitHub Education]
### Hi everyone,

I have not received my education benefits within 5 days of my approval status.  I was approved on September 24, 2024.  Please help!  Thank you.

**Comment by davevad93** [comment]
Hi @sigfriedherzog001 ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @sigfriedherzog001 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @sigfriedherzog001 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

**Comment by sigfriedherzog001** [comment]
I opened three support tickets with JetBrains.  I'm unable to redeam the student license the web site JetBrains and the JetBrains IDE software directs me to GitHub, then using the GitHub links it directs me back to the URL at https://www.jetbrains.com/community/education/?authMethod=github#students

I can't make any progress in asking for help.  The link using the JetBrians IDE software will direct my license request to the URL https://account.jetbrains.com/licenses

This URL then directs me to: https://www.jetbrains.com/shop/eform/students/prolong?data=00871E6C5828F177E0EDA1C53CC39A826C76CD9F81D9E8388E0CED8C39B993E89DFC43E569B60A7979591379591E9DD99CA22FB8E2310431F8DE91CA4711E53CD4001B4B7E51C498B9321AB75722A1D9E7CBC8054247E0EF9C9C6F3EB1B2542928D0C9EEA3AF9EC342C8F34F4BD770A8548D581F66A948D4F12A73D6F1706F716468&_gl=1*qyrgmc*_gcl_au*MTc5MTQ1MDAyMS4xNzI0ODg4Nzkw*_ga*MTc3Njk5MjM0OS4xNzE1MjgyOTA0*_ga_9J976DJZ68*MTcyODM5MjExNC44LjEuMTcyODM5NDI4Mi42MC4wLjA.

This URL then takes me to: https://github.com/education/students
 
 I can't redeem the new license.  I'm asking for help to apply the new license.
 
 Anything you can do is greatly appreciated!
 
 Sigfried Herzog
sigfried.herzog001@mymdc.net
![Screenshot 2024-10-08 at 9 06 48 AM](https://github.com/user-attachments/assets/b2683c2c-ae67-46b4-9ba7-d7c9f3f4ff5b)

**Reply by davevad93** [reply]
Hey @sigfriedherzog001 , try the following:

Go to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section, search for `JetBrains` and click on "***Get access by connecting your GitHub account on JetBrains***". If you can't redeem your student offer by doing that either, contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Comment by sigfriedherzog001** [comment]
Hi Davide,

This is the problem, when I follow the directions, and click on "Get access
by connecting your GitHub account on JetBrains", I get sent back to the
JetBrains webpage at:
https://www.jetbrains.com/community/education/?authMethod=github#students

Like I mentioned before I opened a ticket with with GitHub and they told me
to contact JetBrains.  Below, I'm sharing the communication from GitHub and
they say to open a support ticket with JetBrains.   So, I'm back to asking
for your help to apply the license to my account.  There has to be a way to
fix this problem.

Thank you,

Sigfried Herzog
***@***.***

[GitHub Support] - I'm unable to redeem the new student license using GitHub
External
Inbox
Search for all messages with label Inbox
Remove label Inbox from this conversation
GitHub ***@***.***>
9:20 AM (16 minutes ago)
to me
## Please do not write below this line ##

Your request (Ticket 3039983 <https://support.github.com/ticket/3039983>)
has been updated.

To review the status of your request and add additional comments, please
visit: Ticket 3039983 <https://support.github.com/ticket/3039983>.

*GitHub* (GitHub Support)

Oct 8, 2024, 1:20 PM UTC

Hi Sigfried,

Sorry you are having trouble.

You can access your GitHub Education benefits here:
https://education.github.com/pack/offers

Some of our partner offers are single-use and non-renewable. Most timed
offers begin when redeemed, not when granted.

If you have questions or issues with an offer, please contact the partner
directly by clicking the help available at link on the offer tile:
https://education.github.com/pack/offers

You may wish to include any relevant links or screenshots when you contact
them.

We hope you enjoy your GitHub Education benefits.

*Sigfried Herzog*

Oct 8, 2024, 1:20 PM UTC
<https://enterprise.githubsupport.com/#what-can-github-education-assist-you-with>What
can GitHub Education assist you with?

I am a verified student having trouble redeeming my Global Campus offers
<https://enterprise.githubsupport.com/#description>Description

I'm unable to renew the student pack for JetBrains. I see the message I'm
approved and the license is available, but my JetBrians is linked to my
GitHub, but when I try to get the license, JetBrains will redirect me to
GitHub and then GitHub redirects me to the URL
https://www.jetbrains.com/community/education/?authMethod=github#students

So, I'm stuck in a endless cycle between jetbrains and github with no
ability to apply the student license. Please provide the student license
for ***@***.***
My expiring license is: License ID: HOY7G8KPOS

Thank you!

Sigfried

On Tue, Oct 8, 2024 at 9:26 AM IntelliJ Support <
***@***.***> wrote:

> Hello Sigfried,
>
> Our support team is looking into your request. It was logged as #6856437
> <https://intellij-support.jetbrains.com/hc/requests/6856437>)
>
> We aim to respond within one business day, but it may take longer in some
> cases.
>
> To speed up the investigation, you can upload diagnostic data to our
> server <https://uploads.jetbrains.com/> for analysis:
>
>
>    - Logs: *Help | Collect Logs & Diagnostic Data* or entire folder
>    <https://www.jetbrains.com/help/idea/directories-used-by-the-ide-to-store-settings-caches-plugins-and-logs.html#logs-directory>
>    ;
>    - Demo project where the problem can be reproduced;
>    - Screenshots or videos that illustrate the problem.
>
> You can also check our public bug tracker
> <https://youtrack.jetbrains.com/> and StackOverflow
> <https://stackoverflow.com/>: the problem could be known already and have
> a solution.
>
> To continue the conversation, you can reply to this email or use this
> link: https://intellij-support.jetbrains.com/hc/requests/6856437
> This email is a service from IntelliJ.
>

**Comment by sigfriedherzog001** [comment]
The problem was resolved by logging into GitHub and there was an
obscure URL that had a button to link GitHub to JetBrains...The linking
between GitHub and JetBrains is working and I was able to redeem the
student license.  Thank you for your help, I have not see that link button
until this afternoon.

Here is the confirmation from JetBrains.

Thank you for your assistance!

Sigfried Herzog
***@***.***

License Certificate for your JetBrains Educational Pack / Order D381328408
External
Inbox
Search for all messages with label Inbox
Remove label Inbox from this conversation
[image: image.png]
JetBrains Sales
1:34 PM (4 minutes ago)
to me

Hi Sigfried,

Congratulations! Your JetBrains Educational Pack has been renewed.

Please use your JetBrains Account <https://account.jetbrains.com/> to
access JetBrains tools whenever you want to use them.

And here’s a quick tip. Download the IDE Features Trainer
<https://plugins.jetbrains.com/plugin/8554-ide-features-trainer?utm_source=account&utm_medium=email&utm_campaign=education_auto-reply_student&utm_content=license-certificate-for-your-jetbrains-educational-pack>
plugin
to learn basic shortcuts and essential features from right inside IntelliJ
IDEA and other JetBrains IDEs.

If you have any questions, please email us and we will be glad to help.
------------------------------


*Kind Regards,The JetBrains team*


*www.jetbrains.com <https://www.jetbrains.com/>The Drive to Develop*

**Reply by davevad93** [reply]
Hey @sigfriedherzog001, glad you got it sorted out, have a nice day! 👍
